### ISSUE TITLE
is limit_train_batches shuffle or random

### ISSUE THREAD

**Issue by qmpzzpmq** [PyTorchLightning/pytorch-lightning#2928]
hi, I am using limit_train_batches . If it is set, is it means a subdataset of whole train dataset ? similar with torch.utils.data.random_split

**Comment by ydcjeff** [comment]
Yes, it is a subset of the `train_dataloder`
But, it doesn't similar with `random_split`

**Comment by qmpzzpmq** [comment]
@ydcjeff I mean, is it random?

**Comment by ydcjeff** [comment]
I think it is not random. It is the first `limit_train_batches` of the train dataset.

**Comment by awaelchli** [comment]
Yes exactly, @ydcjeff is right. It will fetch batches from the dataloader until it reaches that amount, so your dataset and dataloader settings regarding shuffling will be respected.

**Comment by qmpzzpmq** [comment]
@awaelchli @ydcjeff thx

**Comment by adosar** [comment]
> @awaelchli @ydcjeff thx

What if the dataloader uses `shuffle == True`?
